src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+
+  const noSpacesArray = sourceString.split(';');
+  const arrayOfArrays = noSpacesArray
+    .map((element) => element.split(':'))
+    .filter((element) => element.length === 2)
+    .map((element) => [element[0].trim(), element[1].trim()]);
```

Comment:
Consider combining the map and filter operations to improve code readability and performance. You can perform the filtering logic within the map operation.